frontend/passkeys.js

```diff
@@ -0,0 +1,35 @@
+document.addEventListener('DOMContentLoaded', function() {
+    'use strict';
+
+    const registerPasskeyButtons = document.querySelectorAll('#register-passkey');
+
+
+    async function registerPasskey() {
+        const response = await fetch('/passkey/registration-options') //fetch PublicKeyCredentialCreationOptions as JSON string
+        const publicKeyCredentialCreationOptionsJSON = await response.json() // convert to JSONObject
+        const credentialCreationOptions = PublicKeyCredential.parseCreationOptionsFromJSON(publicKeyCredentialCreationOptionsJSON); // convert to PublicKeyCredentialCreationOptions
+        const publicKeyCredential = await navigator.credentials.create({publicKey: credentialCreationOptions}); // create PublicKeyCredential
+        console.log('publicKeyCredential: ', publicKeyCredential);
```

Comment:
I don't think there's any need to log anything here unless you think it's useful.  If the public key isn't created properly or is invalid the `register` call will fail anyway